.devcontainer/Dockerfile

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 The Crossplane Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ====================================================================================
+
+FROM golang:1.13-buster
```

Comment:
what's the maintenance burden of this file over time? Will we need to check in on it every so often, or wait till someone says it's not working for them, or how do you see that going?

Reply:
I don’t really have enough time using dev containers to answer that. I do believe we can add it to the CICD pipeline and automate its validation via GitHub actions.

On the other hand, one thing that is holding me back to proceed with this change is that each repo needs to have its own dev container. It can be a Git module but then it becomes too much overhead IMO.

The upgrade to Go 1.14 is the right thing rondo if we can support 1.13 as well.